libr/main/radare2.c

```diff
@@ -1196,6 +1196,11 @@ R_API int r_main_radare2(int argc, char **argv) {
 				}
 			}
 		}
+		if (o && compute_hashes) {
```

Comment:
Why moving this here?

Reply:
Previously, `r_bin_file_hash` requires the `iod` argument which is now no longer needed, so I thought it is best to move it outside the `if (iod && !strstr (iod->uri, "://"))`.

Reply:
Now, i see the problem. Currently, in a debugging session the `iod` will have a size of `-1`. Every time users open r2 in debug mode `-d` they will see the warnings about the hashlimits now. Even if putting the statements back into the original if-else, users will see the warnings when they try to do `it` `itj`.